packages/material-ui-styled-engine-sc/README.md

```diff
@@ -0,0 +1,15 @@
+# @material-ui/styled-engine-sc
+
+This package is a simple wrapper around the `styled-components` package. It is created to be used as alias for the `@material-ui/styled-engine` package, for the developers who would like to use `styled-components` as a styled engine instead of `@emotion/styled`.
```

Comment:
```suggestion
This package is a wrapper around the `styled-components` package. It is created to be used as alias for the `@material-ui/styled-engine` package, for the developers who would like to use `styled-components` as a styled engine instead of `@emotion/styled`.
```